calendar-bundle/src/Resources/contao/languages/en/tl_calendar.xlf

```diff
@@ -149,6 +149,24 @@
       <trans-unit id="tl_calendar.feeds.1">
         <source>Manage RSS feeds</source>
       </trans-unit>
+      <trans-unit id="tl_calendar.breadcrumbMode.0">
```

Comment:
Same here. We should try to re-use labels wherever possible.

Reply:
Obsolute now, because I removed setting as discuessed.